### ISSUE TITLE
Not possible to select "custom format" for Created and modified fields

### ISSUE THREAD

**Issue by lyquix-owner** [FLEXIcontent/flexicontent-cck#614]
When selecting "Custom format" in Created field the option is reverted back to value DATE_FORMAT_LC1. This is happening because the option "Custom format" has a null value, and the XML sets the value DATE_FORMAT_LC1 as default.

Should the value of "Custom format" be changed to a string, such as "custom"?

**Comment by ggppdk** [comment]
aaahhh, i had seen and forgot to make a note or an issue

Fix is small,
will fix today

**Comment by ggppdk** [comment]
```
<field name="date_format" type="multilist" subtype="list" toggle_related="1" default="DATE_FORMAT_LC1" label="FLEXI_FIELD_DATE_FORMAT" description="FLEXI_FIELD_CUSTOM_FORMAT_J16GE_DESC">
```

See the above it has a default value
**default="DATE_FORMAT_LC1"** 

it is not needed, and gets loaded on every form load

will remove it, and issue will be fixed without any break (PHP code has a default value !)

````
<field name="date_format" type="multilist" subtype="list" toggle_related="1" default="" label="FLEXI_FIELD_DATE_FORMAT" description="FLEXI_FIELD_CUSTOM_FORMAT_J16GE_DESC">
```

**Comment by ggppdk** [comment]
Same change needed for "Modified by" field

**Comment by ggppdk** [comment]
Fixed with #9e25e13

**Comment by lyquix-owner** [comment]
Thank you! That was easy :)
